README.md

```diff
@@ -54,7 +54,7 @@ Currently, `com.broadinstitute.dsp.zombieMonitor.DbReaderSpec` and `com.broadins
 
 Once a PR is merged, there will be a PR created in [terra-helm](https://github.com/broadinstitute/terra-helm). 
 Get this PR merged, and another automatic commit will bump leonardo's chart version. This will trigger another automatic commit 
-in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`,
-create another PR for bumping all other environments when you're ready (similar to [this](https://github.com/broadinstitute/terra-helmfile/pull/390)). Once all these PRs are merged, 
-go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click `SYNC APPS` button on the left upper corner for all leonardo deploys (select `PRUNE` option as well).
-you can also choose to `sync` individual environments if you choose to), this will sync leonardo's deployment to match [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) repo.
+in [terra-helmfile](https://github.com/broadinstitute/terra-helmfile), note this commit will only auto bump `dev` and `perf`, and will be auto-merged. Once the terra-helmfile PR is auto-merged, go to [argo](https://ap-argocd.dsp-devops.broadinstitute.org/applications) (you need to be on VPN to access argo), and click `SYNC APPS` button on the left upper corner for dev and perf leonardo deploys (select `PRUNE` option as well).
+this will sync leonardo's deployment to match [terra-helmfile](https://github.com/broadinstitute/terra-helmfile) repo.
+create another PR for bumping all other environments when you're ready (similar to [this](https://github.com/broadinstitute/terra-helmfile/pull/390)) and then after that is merged, sync those environments in argo as well.
```

Comment:
now that I think about it, maybe just mention that syncing to other environments should be left to automation?

Reply:
do we need to verify this first after the next leo release?

Reply:
sure...sounds goood